lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -119,6 +121,29 @@ def get_thread_created_event_data(thread, followed):
     }
 
 
+def get_comment_created_event_name(comment):
+    """Get the appropriate event name for creating a response/comment"""
+    return COMMENT_CREATED_EVENT_NAME if comment.get("parent_id") else RESPONSE_CREATED_EVENT_NAME
+
+
+def get_comment_created_event_data(comment, commentable_id, followed):
+    """
+    Get the event data payload for comment creation (excluding fields populated
+    by track_forum_event)
+    """
+    event_data = {
+        'discussion': {'id': comment.thread_id},
+        'commentable_id': commentable_id,
+        'options': {'followed': followed},
```

Comment:
Not in the UI, but apparently the old API allows it. I'm not sure what effect it has in the CS.